src/components/vanilla/charts/PieChart/index.tsx

```diff
@@ -52,13 +52,47 @@ type Props = {
 type Record = { [p: string]: string };
 
 export default (props: Props) => {
-  const { results, title, enableDownloadAsCSV } = props;
+  const { results, title, enableDownloadAsCSV, maxSegments, metric, slice, onClick } = props;
+
+  const chartRef = useRef<ChartJS>(null);
+
+  const fireClickEvent = (element: InteractionItem[]) => {
+    if (!element.length) {
+      //clicked outside pie
+      onClick({ slice: null, metric: null });
```

Comment:
@tagspace your thinking here is that they would click outside the pie to clear the selection?